checkstyle.xml

```diff
@@ -44,6 +46,9 @@
 			<property name="tokens" value="COMMA"/>
 		</module>
 		<module name="UnusedImports"/>
+		<module name="AvoidStarImport">
+			<property name="allowStaticMemberImports" value="true"/>
```

Comment:
Can you maybe make this to check only if there are 10 or more static imports? As we have defined in checkstyle

Reply:
We don't have that defined in checkstyle, only in the intellij settings. To my knowledge, there's no way to limit this module to below or above any specific number of imports.

Reply:
Yeah I meant code style: `By changing Names count to use static imports with ''*' to 10 (static imports for f.e. ItemIDs with wildcards are preferred)`. But if it is not possible it is fine I guess